src/xhydro/modelling/hydrological_modelling.py

```diff
@@ -333,6 +333,110 @@ def _is_rate(u):
                     f.write(f"{k}; {v}\n")
             ds.to_netcdf(Path(save_as).with_suffix(".nc"), **kwargs)
 
+    elif model == "HBVEC":
+        if not all(
+            v in ds
+            for v in [
+                "rlon",
+                "rlat",
+                "lon",
+                "lat",
+                "orog",
+                "time",
+                "tasmax",
+                "tasmin",
+                "pr",
+            ]
+        ):
+            missing = {
+                "rlon",
+                "rlat",
+                "lon",
+                "lat",
+                "orog",
+                "time",
+                "tasmax",
+                "tasmin",
+                "pr",
+            }.difference(ds.variables)
+            raise ValueError(
+                f"The dataset is missing the following required variables for Raven HBVEC: "
+                f"{missing}."
+            )
+
+        # Remove unused variables
+        ds = ds.drop_vars(
+            set(ds.data_vars)
+            - {"rlon", "rlat", "lon", "lat", "orog", "time", "tasmax", "tasmin", "pr"}
+        )
+
+        # Convert units
+        if _is_rate(ds["pr"].attrs.get("units", "")):
+            ds["pr"] = xc.units.rate2amount(ds["pr"])
+        ds["pr"] = xc.units.convert_units_to(ds["pr"], "mm", context="hydro")
+        ds["tasmax"] = xc.units.convert_units_to(ds["tasmax"], "degC")
+        ds["tasmin"] = xc.units.convert_units_to(ds["tasmin"], "degC")
+        ds = xs.utils.change_units(ds, {"tasmin": "degC", "tasmax": "degC"})
+        ds["orog"] = xc.units.convert_units_to(ds["orog"], "m")
+
+        # Ensure that longitude is in the range [-180, 180]
+        # This tries guessing if lons are wrapped around at 180+ but without much information, this might not be true
+        if np.min(ds["lon"]) >= -180 and np.max(ds["lon"]) <= 180:
+            pass
+        elif np.min(ds["lon"]) >= 0 and np.max(ds["lon"]) <= 360:
+            warnings.warn(
+                "Longitude values appear to be in the range [0, 360]. They will be converted to [-180, 180]."
+            )
+            with xr.set_options(keep_attrs=True):
+                ds["lon"] = ds["lon"] - 180
+
+        # Reorder dimensions to match Raven's expectations for .rvt (x, y, t)
+        # Raven is faster with gridded inputs than with stations when there are a lot of stations
+        ds = ds.transpose("rlon", "rlat", "time")
+
+        # Convert calendar
+        # FIXME: xscen 0.9.1 still calls the old xclim function. This will be fixed in the next release.
+        if Version(xs.__version__) > Version("0.9.10"):
+            convert_calendar_kwargs = {"calendar": "standard", "use_cftime": False}
+        else:
+            convert_calendar_kwargs = {"target": "default"}
```

Comment:
En fait, on devrait changer ça plus haut aussi.
```suggestion
        # Convert calendar
        convert_calendar_kwargs = {"calendar": "standard", "use_cftime": False}
```